### ISSUE TITLE
Website instructions

### ISSUE THREAD

**Issue by timiimit** [timiimit/UT4MasterServer#66]
There needs to be a web page for instructions on how to make the game (and server) connect to the master server. 3 pages are needed. It would probably be best to have links to these in the dropdown on a header of the page.

Related to #8

## Instructions to follow when using stock UT4

- Go to `...\Documents\UnrealTournament\Saved\Config\WindowsNoEditor\` in file explorer
- Open Engine.ini file
- Append the following at the bottom of the file:
```
[OnlineSubsystemMcp.BaseServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.GameServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.AccountServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.OnlineFriendsMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.PersonaServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.OnlineImageServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
```
- Go to `<ut4_install_location>\Engine\Binaries\Win64\`
- Create a shortcut to `UE4-Win64-Shipping.exe` on your desktop
- Open properties of the shortcut
- in textbox target at the very end add ` UnrealTournament -epicapp=UnrealTournamentDev -epicenv=Prod -EpicPortal`
- **make sure there is space between what was in there before and what you added**
- use the shortcut to start the game
- when prompted to log in, enter username instead of email

## Instructions to follow when using UT4 with UT4UU

- Open the game
- open console
- enter command `uu settings`
- go to *Other* tab
- scroll to the bottom
- in the *Master Server Domain* section locate a textbox
- enter `master-ut4.timiimit.com` into the textbox
- press APPLY in bottom right of the window
- close the game
- start game
- follow instructions in UT4UU's login window to add a new account
- Use the newly added account
 
## Instructions to follow for hub owners

- Go to `<ut4_server_install_location>\UnrealTournament\Saved\Config\WindowsServer\` in file explorer
- Open `Engine.ini` file
- Append the following at the bottom of the file:
```
[OnlineSubsystemMcp.BaseServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.GameServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.AccountServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.OnlineFriendsMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.PersonaServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
[OnlineSubsystemMcp.OnlineImageServiceMcp]
Domain=master-ut4.timiimit.com
Protocol=https
```
- Start the server as you normally would

**Comment by caseyswilliams** [comment]
I'll take care of this tomorrow

**Comment by timiimit** [comment]
Closed by #69
